git_info/cli.py

```diff
@@ -125,6 +126,26 @@ def has_changes(work_dir, debug):
     click.echo(f"::set-output name=has_changes::{'true' if repo.is_dirty() else 'false'}")
 
 
+@run.command(name="last-commit")
+@click.option("--days", default=os.getenv("LAST_COMMIT_DAYS", 10), required=True, type=int)
+@click.option("--debug", is_flag=True)
+def last_commit(debug, days):
+    """
+    Checks remotely if any commit has been pushed in last X days (default: 10)
+    """
+    token = os.getenv('GITHUB_TOKEN')
+    repo_name = os.getenv("GITHUB_REPOSITORY")
+    ref = os.getenv("GITHUB_REF")
+    headers = {"Authorization": f"bearer {token}"}
```

Comment:
Just a personal taste, I would just remove `headers` variable and use this dict directly in `requests.get`